airflow/providers/google/cloud/transfers/postgres_to_gcs.py

```diff
@@ -112,7 +112,13 @@ def __init__(
         self.cursor_itersize = cursor_itersize
 
     def _unique_name(self):
-        return f"{self.dag_id}__{self.task_id}__{uuid.uuid4()}" if self.use_server_side_cursor else None
+        """
+        Generates a deterministic UUID for the cursor name,
+        using the combination of DAG ID and task ID.
+        """
+        if self.use_server_side_cursor:
+            return str(uuid5(uuid5(NAMESPACE_OID, self.dag_id), self.task_id))
```

Comment:
Hmm. You are right - but maybe it's enough to just use UUID1 and we are done. I don't think we need dag_id, task_id, the probability of getting same uuid is 1/ 2^128 = 3.4028237e+38

Reply:
dag_id abd task_id might be useful for some monitoring tools around postgres for better identify, uuid4 here just for some uniqueness.  

I'm feeling not good about uuid1 because it expose MAC address, which might not be so bad in this situation.

uuid7 still in draft, or at least it not supported in current version of Python, I'm not check it status for couple years.

But yeah it could be any random/random-like/time-based staff which fit into the 63 bytes, it is really small chance that someone recompile postgres with reduce allowed size of all identifiers.